src/main/java/org/gridsuite/modification/server/modifications/VscCreation.java

```diff
@@ -176,13 +176,16 @@ private VscConverterStation createConverterStation(Network network,
                                                        ConverterStationCreationInfos converterStationCreationInfos,
                                                        ReportNode subReportNode,
                                                        String logFieldName) {
-        ReportNode converterStationReporter = subReportNode.newReportNode().withMessageTemplate("converterStationCreated" + logFieldName, logFieldName).add();
+        ReportNode converterStationReporter = subReportNode.newReportNode()
+            .withMessageTemplate("converterStationCreated", "New converter station with id=${id} created")
+            .withUntypedValue("id", converterStationCreationInfos.getEquipmentId())
+            .add();
         VoltageLevel voltageLevel = ModificationUtils.getInstance().getVoltageLevel(network, converterStationCreationInfos.getVoltageLevelId());
         VscConverterStation vscConverterStation = voltageLevel.getTopologyKind() == TopologyKind.NODE_BREAKER ?
                 createConverterStationInNodeBreaker(network, voltageLevel, converterStationCreationInfos, converterStationReporter) :
                 createConverterStationInBusBreaker(voltageLevel, converterStationCreationInfos, converterStationReporter);
         converterStationReporter.newReportNode()
-                .withMessageTemplate("converterStationCreated" + logFieldName, "New converter station with id=${id} created")
+                .withMessageTemplate("converterStationCreated" + logFieldName, logFieldName + " with id=${id} created :")
```

Comment:
```suggestion
                .withMessageTemplate("converterStationCreatedInfo", "${fieldName} with id=${id} created :")
                .withUntypedValue("fieldName", logFieldName)
```
In my opinion, if generating dynamically the key with logFieldName, we can not reuse the template. Key should be static and is only different to other key if not the same template.

In similarity, template should be static also, use parameterized template instead of creating dynamically template